specification/columns/relatedresourceid.md

```diff
@@ -0,0 +1,31 @@
+# Related Resource ID
```

Comment:
This assumes there can only ever be a single relationship. This feels like it won't scale. We could add it as a JSON object, but it seems like we'd need to put more thought into it before going that route.